components/dashboard/src/teams/Members.tsx

```diff
@@ -235,7 +235,11 @@ export default function () {
                                     </div>
                                 </ItemField>
                                 <ItemField className="my-auto">
-                                    <span className="text-gray-400">{dayjs(m.memberSince).fromNow()}</span>
+                                    <Tooltip className="w-fit" content={dayjs(m.memberSince).format("MMM D, YYYY")}>
+                                        <span className="text-gray-400 hover:text-gray-200">
+                                            {dayjs(m.memberSince).fromNow()}
+                                        </span>
+                                    </Tooltip>
```

Comment:
issue: Did we change any hover style on the members page?

| Members (Default) | Members (Hover) |
|-|-|
| <img width="1440" alt="Screenshot 2023-01-24 at 12 09 30 AM (2)" src="https://user-images.githubusercontent.com/120486/214161118-4d772b06-6de1-4aa8-85d4-1f8b9a8d81cc.png"> | <img width="1440" alt="Screenshot 2023-01-24 at 12 09 32 AM (2)" src="https://user-images.githubusercontent.com/120486/214161128-338297a0-7240-48d8-b78d-efba3186870c.png"> |

Reply:
Looks good—resolving.